airflow/providers/docker/operators/docker_swarm.py

```diff
@@ -86,8 +86,16 @@ class DockerSwarmOperator(DockerOperator):
     :param mode: Indicate whether a service should be deployed as a replicated or global service,
         and associated parameters
     :param networks: List of network names or IDs or NetworkAttachmentConfig to attach the service to.
-    :param placement: Placement instructions for the scheduler. If a list is passed instead,
-        it is assumed to be a list of constraints as part of a Placement object.
+        NetworkAttachmentConfig object accepts following params:
+        :param target: (str) - The target network for attachment. Can be a network name or ID.
+        :param aliases: (list) - A list of discoverable alternate names for the service.
+        :param options: (dict) - Driver attachment options for the network target.
```

Comment:
These can’t use `:param` since it does not support nesting (as far as I know). You need to use simple bullet points.

Reply:
I see. I will update it @uranusjr